less/mixins.less

```diff
@@ -1 +1,21 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.firstbutton(@color, @pad, @background, @size, @radius) {
```

Comment:
Since this mixin can be reused, the name is a bit confusing. Consider a more general name